recipe/meta.yaml

```
@@ -1,47 +1,60 @@
 {% set name = "recordlinkage" %}
-{% set version = "0.14" %}
+{% set version = "0.16" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-
-  sha256: 92e636314bb068b6f6dbc93090998e9e753e3a895972f1a584a4d389a99af93e
-
+  sha256: ecda0c10dff138b1706815de332b1285f670ae7e8cce92596213501d589e6aa4
 
 build:
-  noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  # jellyfish >=1 and maturin aren't available on s390x
+  skip: True  # [s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
-  build:
   host:
-    - python >=3.5
+    - python
     - pip
+    - setuptools
+    - setuptools_scm
+    - wheel
   run:
-    - python >=3.5
-    - numpy
+    - python
+    - jellyfish >=1
+    - joblib
+    - numpy >=1.13
+    # pandas already has numexpr and bottleneck as dependencies
     - pandas >=0.18.0
```

```suggestion
    - pandas >=1,<3
```
https://github.com/J535D165/recordlinkage/blob/v0.16/pyproject.toml#L20C12-L20C18